tests/plugins/test_pin_operator_digests.py

```diff
@@ -402,8 +403,8 @@ def test_orchestrator_exclude_csvs(self, docker_tasker, tmpdir, caplog, has_envs
     def test_orchestrator(self, docker_tasker, tmpdir, caplog):
         pullspecs = [
             # final-registry: do not replace registry or repos
-            'final-registry/ns/foo@sha256:1',  # -> no change
-            'final-registry/ns/foo:1',  # -> final-registry/ns/foo@sha256:1
+            'registry.private.example.com/ns/foo@sha256:1',  # -> no change
+            'registry.private.example.com/ns/foo:1',  # -> final-registry/ns/foo@sha256:1
```

Comment:
`final-registry` still mentioned in comments

Reply:
renamed